### ISSUE TITLE
Unhandled group messages kill sending actor.

### ISSUE THREAD

**Issue by alatdneg** [actor-framework/actor-framework#1111]
I've got actors joining a group and some don't handle all the messages that can be received from it.

When the receiver prints and drops, the sender then becomes unresponsive.

receiver
*** unexpected message [id: 80, name: PlayheadSelectionActor]: (caf::event_atom(), caf::change_atom())

Something tries to talk to the sender.
[2020-06-04 10:31:41.070] [xstudio] [warning] xstudio::ui::qml::PlaylistUI::update_media()::<lambda(const caf::error&)> system_error(request_receiver_down)
[2020-06-04 10:31:41.070] [xstudio] [warning] xstudio::ui::qml::PlaylistUI::update_playheads()::<lambda(const caf::error&)> system_error(request_receiver_down)
[2020-06-04 10:31:41.484] [xstudio] [warning] Move to failed system_error(request_receiver_down).
[2020-06-04 10:31:41.510] [xstudio] [warning] Move to failed system_error(request_receiver_down).

It looks like if any subscriber to the group fails to handle messages sent to the group the original sender is killed in someway, like it's getting some error sent back to is from the receiver.

Currently I can get around this by handling every possible message sent to the group, but it's bit of a pain.

**Comment by uentity** [comment]
You're right about the source of this issue -- when actor can't handle incoming message, standard `print_and_drop` handler will send back an error message. Sender receives that message and dies,

What you need to do is install custom error handler for sender. See related question here: https://groups.google.com/forum/#!topic/actor-framework/qQBoNwkRob4
